tests/PluginTest.php

```diff
@@ -35,8 +35,30 @@ public function testGetSubscribedEvents()
         $this->assertInternalType('array', $plugin->getSubscribedEvents());
     }
 
+    /**
+    * Tests handleTableflipCommand().
+    */
     public function testHandleTableflipCommand()
     {
+        $channels = "#channel1,#channel2";
+        $plugin = new Plugin;
+        $event = $this->getMockCommandEvent();
+        $queue = $this->getMockEventQueue();
+
+        Phake::when($event)->getSource()->thenReturn('#channel1');
+        Phake::when($event)->getCommand()->thenReturn('PRIVMSG');
+
+        $plugin->handleTableflipCommand($event, $queue);
+        Phake::when($event)->getCustomParams()->thenReturn(array($channels));
+
+        $plugin->handleTableflipCommand($event, $queue);
+        Phake::when($event)->getCustomParams()->thenReturn(array($channels,'!tableflip','I','literally','just','ate','a','cricket','right','now'));
+
+        Phake::inOrder(
+            Phake::verify($queue, Phake::atLeast(1))->ircPrivmsg('#channel1', $this->isType('string')),
+            Phake::verify($queue)->ircPrivmsg('#channel1', '(╯°□°）╯︵ ┻━┻  ʍou ʇɥƃıɹ ʇǝʞɔıɹɔ ɐ ǝʇɐ ʇsnɾ ʎꞁꞁɐɹǝʇıꞁ I')
```

Comment:
here was 1 failure:

1) Phergie\Irc\Tests\Plugin\React\TableFlip\PluginTest::testHandleTableflipCommand
Failed asserting that Expected Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(equal to <string:#channel1>, equal to <string:(╯°□°）╯︵ ┻━┻  ʍou ʇɥƃıɹ ʇǝʞɔıɹɔ ɐ ǝʇɐ ʇsnɾ ʎꞁꞁɐɹǝʇıꞁ I>) to be called exactly <1> times, actually called <0> times.
Other Invocations:
  Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(<string:#channel1>, <string:Usage: tableflip message>)
  Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(<string:#channel1>, <string:message - the message to flip and then send (all words after this are assumed to be part of message)>)
  Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(<string:#channel1>, <string:Instructs the bot to return a flipped table and the inverted words in the message.>)
  Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(<string:#channel1>, <string:(╯°□°）╯︵ ┻━┻ >)
  Phergie\Irc\Bot\React\EventQueueInterface->ircPrivmsg(<string:#channel1>, <null>).

/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/vendor/phake/phake/src/Phake/Client/PHPUnit.php:56
/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/vendor/phake/phake/src/Phake/Proxies/VerifierProxy.php:109
/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/tests/PluginTest.php:59
/var/www/html/phergie/phergie/scaffold/phergie-irc-plugin-react-tableflip/tests/PluginTest.php:59